student.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'person'
+
+class Student < Person
```

Comment:
- Great work creating the has-many relationship in your classroom class. Now let's create the belongs to the classroom on the student class.

- [ ] It will be great to ensure you create a  method that serves as the 'belongs-to' side (a student belongs to a classroom) in this class. You can see the link to the lesson on this [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)